appcues/src/main/java/com/appcues/trait/appcues/PagingDotsTrait.kt

```diff
@@ -69,13 +70,9 @@ internal class PagingDotsTrait(
 
         Box(
             modifier = Modifier
+                .padding(wrapperInsets)
```

Comment:
here we see paging dots needs to apply the safe area, so it uses as a padding modifier on the root box